### DISCUSSION TITLE
Line height lazyvim

### DISCUSSION THREAD

**Discussion by ashikpy** [LazyVim/LazyVim#4413] [General]
Im wondering about how to change the line-height of the lines in nvim

**Comment by yusufaktepe** [comment]
It depends on which terminal you're using. You should look for how to set line-height in your terminal.

**Reply by ashikpy** [reply]
i did, use my terminal's line-height

but it ends up staggering

<img width="785" alt="image" src="https://github.com/user-attachments/assets/50e37949-a16e-4d65-a3cf-b0acf72ceb72">

**Reply by yusufaktepe** [reply]
This is how it looks in Kitty (with `modify_font cell_height 8px`). My guess is that it's a terminal or font issue.

![image](https://github.com/user-attachments/assets/f21e6b20-a308-44ee-96df-01a54346b4ad)

**Reply by ashikpy** [reply]
i switched to tabby, so no problem now i guess.
